crates/bevy_tasks/src/task_pool.rs

```diff
@@ -309,6 +336,42 @@ mod tests {
         assert_eq!(count.load(Ordering::Relaxed), 100);
     }
 
+    #[test]
+    fn test_spawn_pollable() {
+        let transform_fn = |i: usize| i * 3;
+
+        let pool = TaskPool::new();
+        let nums: Vec<usize> = (1..10).into_iter().collect();
```

Comment:
Afaik, `into_iter` is unneeded here. I.e. 
```suggestion
        let nums: Vec<usize> = (1..10).collect();
```
would be just as valid.